TestON/drivers/common/cli/emulator/remotemininetdriver.py

```diff
@@ -376,9 +376,20 @@ def disconnect( self ):
         if self.handle:
             # Close the ssh connection
             self.handle.sendline( "" )
-            self.handle.expect( "\$" )
-            self.handle.sendline( "exit" )
-            self.handle.expect( "closed" )
+            #self.handle.expect( "\$" )
+            i = self.handle.expect( [ '\$', 'mininet>', pexpect.TIMEOUT ],
```

Comment:
Right, I am using optical.py script to start a packet layer topology for packet optical use case. The reason I can't use mininetclidriver's start net() is because this function takes the path of the topo file and name of the topi as arguments. The optical.py script provided by packet optical team starts the topology in the script itself (no command line required)and therefore cannot use start net() because this functions takes the argument and then spins the topology from the command line.